src/util/std_expr.h

```diff
@@ -3897,6 +3897,21 @@ class member_exprt:public unary_exprt
   {
     return op0();
   }
+
+  static void check(
```

Comment:
The two functions you introduce should be documented

Reply:
@smowton Still to be done?

Reply:
Is there really any value in documenting one-liner that says "assert(ops.count == 1)"?

Reply:
I'm not so sure, I was just confused by you giving a thumbs-up but then seemingly no such change was included.

Reply:
I documented `validate`, which is much more substantial